dags/pull_weather.py

```diff
@@ -0,0 +1,92 @@
+"""
+Pipeline to pull weather prediction data from Envrionment Canada and upsert into weather.prediction_daily table.
+Then, 
+A Slack notification is raised when the airflow process fails.
+"""
+import sys
+from airflow import DAG
+from datetime import datetime, timedelta
+from airflow.operators.python_operator import PythonOperator
+from airflow.hooks.base_hook import BaseHook
+from airflow.contrib.operators.slack_webhook_operator import SlackWebhookOperator
+from airflow.hooks.postgres_hook import PostgresHook
+from airflow.models import Variable
+from dags.pull_wys import SLACK_CONN_ID 
```

Comment:
we don't need this line `from dags.pull_wys import SLACK_CONN_ID `. `SLACK_CONN_ID ` is already defined below